main/FatesRestartInterfaceMod.F90

```diff
@@ -1568,15 +1724,15 @@ subroutine define_restart_vars(this, initialize_variables)
          units='kg/m2/yr', flushval = flushzero, &
          hlms='CLM:ALM', initialize=initialize_variables, ivar=ivar, index = ir_emanpp_si)
     
-   call this%DefineRMeanRestartVar(vname='fates_tveg24patch',vtype=cohort_r8, &
-        long_name='24-hour patch veg temp', &
-        units='K', initialize=initialize_variables,ivar=ivar, index = ir_tveg24_pa)
+    call this%DefineRMeanRestartVar(vname='fates_tveg24patch',vtype=cohort_r8, &
+         long_name='24-hour patch veg temp', &
+         units='K', initialize=initialize_variables,ivar=ivar, index = ir_tveg24_pa)
 
-   call this%DefineRMeanRestartVar(vname='fates_disturbance_rates',vtype=cohort_r8, &
-        long_name='disturbance rates by donor land-use type, receiver land-use type, and disturbance type', &
-        units='1/day', initialize=initialize_variables,ivar=ivar, index = ir_disturbance_rates_siluludi)
+    call this%DefineRMeanRestartVar(vname='fates_disturbance_rates',vtype=cohort_r8, &
+         long_name='disturbance rates by donor land-use type, receiver land-use type, and disturbance type', &
+         units='1/day', initialize=initialize_variables,ivar=ivar, index = ir_disturbance_rates_siluludi)
 
-   if ( hlm_regeneration_model == TRS_regeneration ) then
+    if ( hlm_regeneration_model == TRS_regeneration ) then
```

Comment:
Whitespace changes here and above can be reverted (and should, to preserve helpful git blame history).